src/ORM/DataList.php

```diff
@@ -1447,8 +1497,17 @@ private function fillEmptyEagerLoadedRelations(
      * You can specify nested relations by using dot notation, and you can also pass in multiple relations.
      * When specifying nested relations there is a maximum of 3 levels of relations allowed i.e. 2 dots
      *
-     * Example:
+     * Examples:
+     * <code>
      * $myDataList->eagerLoad('MyRelation.NestedRelation.EvenMoreNestedRelation', 'DifferentRelation')
+     * </code>
```

Comment:
Adding `<code>` tags here is an unrelated improvement for API docs since I'm adding another example here.